turbopack/crates/turbopack-core/src/source_map/mod.rs

```diff
@@ -313,18 +320,30 @@ impl SourceMap {
     /// synthetic code or user-authored original code.
     #[turbo_tasks::function]
     pub async fn lookup_token(self: Vc<Self>, line: usize, column: usize) -> Result<Vc<Token>> {
-        let token = match &*self.await? {
+        Ok(self.lookup_token_and_source(line, column).await?.token)
```

Comment:
Maybe a lookup_token_and_source_internal method with a `receive_source_content: bool` flag to enable/disable getting the source content